### PR TITLE
Add Tizen Studio 2.5 Release Notes

### PR DESCRIPTION
**Pull request by kvbhat**
Signed-off-by: Karthik Bhat <kv.bhat@samsung.com>

### Change Description ###

Describe your changes here.
Adding Tizen Studio 2.5 Release Notes

### CODE REVIEW

**Comment by mijin85cho** [comment]
Target publish date on TD : 8.7
Hello, @annie-samsung. Please take care of this PR

Hello, @kvbhat.
When you fix the PR according to SRI-B's review comment, please update docs/application/tizen-studio/release-notes/release-notes.md file, too.
Reference link : https://github.com/Samsung/tizen-docs/pull/282

File: docs/application/tizen-studio/release-notes/2-5-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+-   Installer  and Package Manager
+	-   Fix extension download failure when url starts with https.
+	-   Fixed a issue of uninstaller window getting closed while uninstallation.
+	-   Fixed a issue of target directory not being able to set after invalid path.
+-   Base IDE
+	-   Supporting "Show in History" view of eclipse team service. 
+-   CLI
+	-   Fixed CLI Build failure for IoT Headless App.
+	> **Note**
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-5-release-notes.md]: Note tag style:
> **Note**
>
> If you have projects with old CLI build scripts, you must re-create the build scripts by running *export to CLI* before running the CLI commands.

It is recommended to use *export to CLI* if the purpose is to emphasize.
I changed Export to export, please verify.

File: docs/application/tizen-studio/release-notes/2-5-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	-   Integrated Certificate Manager into RT IDE
+-   Baseline SDK Size Optimizations
+	-   Optimized Baseline SDK size from 480MB to 320MB
+
+### Fixed Bugs
+-   Certificate Manager
+	-   Added validation of .pem & .der files while importing as a new IoT profile in Certificate Manager
+
+-   Installer  and Package Manager
+	-   Fix extension download failure when url starts with https.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-5-release-notes.md]: Fixed the extension download failure, which occurs when URL starts with https.

File: docs/application/tizen-studio/release-notes/2-5-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+	-   Fixed a issue of uninstaller window getting closed while uninstallation.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-5-release-notes.md]: Fixed an issue of uninstaller window, earlier the window gets closed while uninstalling.

File: docs/application/tizen-studio/release-notes/2-5-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+	-   Fixed a issue of target directory not being able to set after invalid path.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-5-release-notes.md]: Fixed an issue of target directory, now it is possible to set the target directory after an invalid path.

If we rephrase, the technical accuracy is intact?

File: docs/application/tizen-studio/release-notes/2-5-release-notes.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+	-	In case if you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+-	Common
+	-   If you install the Tizen Studio in a directory that requires administrator privileges for access. For example, for C:\ProgramFiles, administrator privileges are required to run the Tizen SDK tools. The Tizen Installer and the baseline SDK Installer alerts you, if you try to install into such a directory.
+-	Web and Native IDE
+	-   From Tizen Studio 2.0 onwards, the Connection Explorer is replaced with the Device Manager, this can cause errors in the Connection Explorer view. You can fix this in two ways:
+	    -   Reset the perspective.
+            In the Tizen Studio menu, select Window > Perspective > Reset Perspective
+	    -   After updating to the Tizen Studio 2.0, run the *eclipse.exe -clean -clearPersistedState* command. Then launch the Tizen Studio normally.
+	-   You can create unit tests for Tizen 2.3.2 and higher version projects only. Now the Tizen Studio does not support unit testing for older versions.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-5-release-notes.md]: You can create unit tests for Tizen 2.3.2 and later version projects only. Now the Tizen Studio does not support unit testing for earlier versions.

File: docs/application/tizen-studio/release-notes/2-5-release-notes.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+	-   Overwriting a duplicate certificate profile in the migration wizard works incorrectly on macOS.
+	-   IoT certification currently requires a user to manually download and select the certificate.
+-	Native UI Builder
+	-   If the expanded attribute in a multibutton entry component is set to false, + is displayed.
+-	Native Component Designer
+	-   The vector-type part is not supported. You cannot see the vector image and change the SVG file.
+	-   Playing sound is not supported on Windows&reg; or macOS.
+	-   The Component Designer crashes if an alias is selected as the source group of an added item.
+-	Emulator
+	-   To use the Tizen emulator, install an Intel VTx supported by the CPU, and the latest version of the graphic card driver provided by the vendor. To verify the prerequisites for the Tizen emulator, see [Prerequisites for the Tizen Studio](https://developer.tizen.org/development/tizen-studio/download/installing-tizen-studio/prerequisites).
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-5-release-notes.md]: To use the Tizen emulator, install an Intel VTx supported by the CPU and the latest version of the graphic card driver provided by the vendor.

File: docs/application/tizen-studio/release-notes/2-5-release-notes.md (hunk 1 above)
Comment (mijin85cho) [reply docs/application/tizen-studio/release-notes/2-5-release-notes.md]: Hello, @Reni-90 
As in Markdown document it is hard to recognize the italic font at once, how about using `code tag style` or **bold font**?

Comment (Reni-90) [reply docs/application/tizen-studio/release-notes/2-5-release-notes.md]: If it is a command then it is better to use `code tag style`, or bold font is also fine.

File: docs/application/tizen-studio/release-notes/2-5-release-notes.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+### Fixed Bugs
+-   Certificate Manager
+	-   Added validation of .pem & .der files while importing as a new IoT profile in Certificate Manager
+
+-   Installer  and Package Manager
+	-   Fix extension download failure when url starts with https.
+	-   Fixed a issue of uninstaller window getting closed while uninstallation.
+	-   Fixed a issue of target directory not being able to set after invalid path.
+-   Base IDE
+	-   Supporting "Show in History" view of eclipse team service.
```
Comment (Reni-90) [inline docs/application/tizen-studio/release-notes/2-5-release-notes.md]: Supports **Show in History** view of eclipse team service.

Use bold or italics to emphasize.

**Comment by mijin85cho** [comment]
Hello, @Reni-90 

@kvbhat updated this PR according to your feedback. However, there are some parts he applied his own fix.
Please check the updated fix.


Hello, @kvbhat.
Please don't forget to apply below feedback of mine.

>Hello, @kvbhat.
>When you fix the PR according to SRI-B's review comment, please update docs/application/tizen-studio/release-notes/release-notes.md file, too.
>Reference link : #282

Regards

**Comment by Reni-90** [comment]
@mijin85cho 

The updated changes looks fine. 
According to your suggestion, the [release-notes.md file]( docs/application/tizen-studio/release-notes/release-notes.md file) is also updated.

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**